.github/workflows/release.yaml

```diff
@@ -13,19 +13,14 @@ jobs:
     uses: ./.github/workflows/ci.yaml
     secrets: inherit
 
-  build:
-    name: Build charm
-    uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@v24.0.6
-
   release:
     name: Release charm
     needs:
       - ci-tests
-      - build
-    uses: canonical/data-platform-workflows/.github/workflows/release_charm.yaml@v24.0.6
+    uses: canonical/data-platform-workflows/.github/workflows/release_charm.yaml@v29.0.5
     with:
       channel: dpe/edge
-      artifact-prefix: ${{ needs.build.outputs.artifact-prefix }}
+      artifact-prefix: ${{ needs.ci-tests.outputs.artifact-prefix }}
```

Comment:
Are we now releasing artifacts built outside of our trusted machines.
This opens us to https://en.wikipedia.org/wiki/XZ_Utils_backdoor